### PR TITLE
[lts/6.5][fix/snpe] Handle zero dim models

### PR DESCRIPTION
**Pull request by anyj0527**
- Some snpe model has "0" for their output layer which is same as "1"
- Handle this corner case

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc [hunk 1]
```diff
@@ -589,6 +589,10 @@ snpe_subplugin::configureUserBuffer (zdl::DlSystem::UserBufferMap &buffer_map, c
     std::vector<size_t> strides (bufferShape.rank ());
     strides[strides.size () - 1] = sizeof (float);
     for (size_t i = strides.size () - 1; i > 0; --i) {
+      /* Some model has "0" dim, replacing 0 with 1 */
+      if (bufferShape[i] == 0) {
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc:593]: Is this the SNPE bug you've mentioned yesterday?

Comment (myungjoo) [reply ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc:593]: let's add something like:
```
  static int reported = 0;
  if (G_UNLIKELY(!reported)) {
    reported = 1;
    log info-level (....)
  }
```

Comment (anyj0527) [reply ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc:593]: Right, after replacing 0 with 100, it works. Now I've asked the model making counterpart.

**Review by myungjoo — APPROVED** [review]
LGTM. Proceed when you are ready. And please consider that "one-time" log-info printing.

**Comment by anyj0527** [comment]
@myungjoo This PR is WIP, can it be reverted?

**Pull request merged**